tpm2/tpm2.go

```diff
@@ -309,6 +304,17 @@ func GetCapability(rw io.ReadWriter, capa Capability, count, property uint32) (v
 	return decodeGetCapability(resp)
 }
 
+// GetManufacturer returns the manufacturer ID
+func GetManufacturer(rw io.ReadWriter) ([]byte, error) {
+	caps, _, err := GetCapability(rw, CapabilityTPMProperties, 1, uint32(Manufacturer))
+	if err != nil {
+		return nil, err
+	}
+
+	prop := caps[0].(TaggedProperty)
```

Comment:
Should we sanity-check `len(caps) >= 1` here?

Reply:
So if GetCapablity succeeds, it should always return at least the number of properties requested. In theory, the TPM could just do the wrong thing, but in that case, panicking seems fine.

We could also just add a check for this in GetCapability itself, but that seems like it should be in another PR.